### ISSUE TITLE
Will the extension be split up?

### ISSUE THREAD

**Issue by Maddremor** [lensvol/fl-small-mercies#21]
Old news but my mind and attention wandered to the [Automation, Extension, Verification](https://www.failbettergames.com/news/automation-extension-verification) post made by FBG, and it seems to me that Small Mercies isn't doing great on the part where extensions should be atomic in design. Are there plans to split up SM functionality to focused extensions? 

(Actually I have ideas for that, but there is no lack of Idea Guys on the internet.)

**Comment by lensvol** [comment]
Yes, from the perspective of the FBG policy mentioned above, Small Mercies is not "atomic" and possibly will never be that. And it is okay from my perspective, as the conditions in the post mentioned above only prevent the extension from being _"whitelisted"_ - not the same thing as _"allowed"_.

I can understand devs' position on this, and at least they finally made it known.

With that in mind, I agree some of the bigger features inside SM (Favour sidebar, Two-Step Confirmation, Currencies) may benefit from being moved into separate extensions.

The main hurdle on that road is that they all share a common codebase, which is responsible for acquiring low-level information from the game, and moving this into a separate library constitutes non-trivial amount of work on my part.

Plans are being made for that, but for now I cannot make any tangible commitments.
